cmd/gh/main.go

```diff
@@ -127,17 +127,12 @@ func main() {
 
 	cs := cmdFactory.IOStreams.ColorScheme()
 
-	authCheckEnabled := os.Getenv("GITHUB_TOKEN") == "" &&
-		os.Getenv("GITHUB_ENTERPRISE_TOKEN") == "" &&
```

Comment:
Is dropping of the environment variables check here intentional? Could it cause any change in functionality?

I had expected that this check gets expanded with `GH_*` variants too.